app/src/main/java/com/uw/fydp/flexeat/flexeat/MenuActivity.java

```diff
@@ -65,22 +71,34 @@ protected void onCreate(Bundle savedInstanceState) {
             mainCourseJSONArray.put(listOfMainCourse.get(i).getJSONObject());
         }
 
+        JSONArray drinksJSONArray = new JSONArray();
+        for (int i = 0 ; i < drinksNames.length ; i++){
+            listOfDrinks.add(new MenuItem(drinksNames[i], false));
+            drinksJSONArray.put(listOfDrinks.get(i).getJSONObject());
+        }
+
+        JSONArray dessertsJSONArray = new JSONArray();
+        for(int i = 0; i < dessertsNames.length; i++){
+            listOfDesserts.add(new MenuItem(dessertsNames[i], false));
+            dessertsJSONArray.put(listOfDesserts.get(i).getJSONObject());
+        }
```

Comment:
maybe can refactor to a generic method that adds MenuItem objects to a list

Reply:
waiting on the format of the response to be finalized and it coming from the backend. so once we have that we can refactor it